fanficfare/adapters/adapter_royalroadcom.py

```diff
@@ -119,9 +119,18 @@ def normalize_chapterurl(self,url):
                 return self.chapterUrls[chapter_url_index]['url']
         return url
 
-    def make_soup(self,data):
+
+		
+    def make_soup(self, data):
         soup = super(RoyalRoadAdapter, self).make_soup(data)
-        self.handle_spoilers(soup)
```

Comment:
Still need to call `handle_spoilers()`

Reply:
fixed that . I missed it cause all the stories I was texting didn't have any